src/components/Home.js

```diff
@@ -1,16 +1,11 @@
-import React, { Component } from "react";
```

Comment:
Not sure what this component was about, but it didn't need to be a class, and now has clear destructured props and implicitly returns JSX! 😍